### PR TITLE
clang: Disable source order sorting for now.

### PR DESCRIPTION
**Pull request by emilio**
This disables (hopefully temporarily, if this is eventually going to be needed)
source order sorting, for causing correctness regressions like #2556, for which
I added a test.

### CODE REVIEW

File: bindgen-tests/tests/headers/issue-2556/LayoutConstants.h [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#include "nsSize.h"
+
+namespace foo {
+
+static constexpr nsSize kFallbackIntrinsicSize(0, 0);
+
+}  // namespace mozilla
```
Comment (reitermarkus) [inline bindgen-tests/tests/headers/issue-2556/LayoutConstants.h:7]: Should this say `foo`?

Comment (emilio) [reply bindgen-tests/tests/headers/issue-2556/LayoutConstants.h]: sure, no reason for the comment really actually, since it's a tiny file.

**Comment by reitermarkus** [comment]
Thanks, I will have a look at the new test.

**Review by pvdrz — APPROVED** [review]

**Comment by pvdrz** [comment]
@emilio now that this is merged, is there any way you can check if there's any other regressions from your side before releasing a new version?

**Pull request merged**